lib/cookie.c

```diff
@@ -97,6 +97,25 @@ Example set of cookies:
 
 static void strstore(char **str, const char *newstr, size_t len);
 
+/* number of seconds in 400 days */
+#define COOKIES_MAXAGE (400*24*3600)
+
+/* Make sure cookies never expire further away in time than 400 days into the
+   future. (from RFC6265bis draft-19)
+
+   For the sake of easier testing, align the capped time to an even minute.
+*/
+static void cap_expires(time_t now, struct Cookie *co)
+{
+  if((TIME_T_MAX - COOKIES_MAXAGE - 30) > now) {
+    timediff_t cap = now + COOKIES_MAXAGE;
+    if(co->expires > cap) {
+      timediff_t remain = cap % 60;
+      co->expires = cap + ((remain > 30) ? (60 - remain) : - remain);
```

Comment:
If `cap == 2000000071`, `remain` will be 31 and `co->expires` will be 2000000100 which is not divisible by 120 (not an even minute). Either the comment above is wrong, or this code is supposed to do something else.

Reply:
By the looks of it, it just rounds `co->expires` to the nearest minute?